Formula/adios2.rb

```diff
@@ -0,0 +1,81 @@
+class Adios2 < Formula
+  desc "Next generation of ADIOS developed in the Exascale Computing Program"
+  homepage "https://adios2.readthedocs.io"
+  url "https://github.com/ornladios/ADIOS2/archive/v2.5.0.tar.gz"
+  sha256 "7c8ff3bf5441dd662806df9650c56a669359cb0185ea232ecb3578de7b065329"
+  head "https://github.com/ornladios/ADIOS2.git", :branch => "master"
+
+  depends_on "cmake" => :build
```

Comment:
If you think building mpi4py with cython is a good idea then would adding this make sense? (And the extra logic required to use cython in the install block.)

```suggestion
  depends_on "cmake" => :build
  depends_on "cython" => :build
```

Or would it require a stand alone mpi4py formula?